decidim-proposals/app/controllers/decidim/proposals/proposal_votes_controller.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    # Exposes the proposal vote resource so users can vote proposals.
+    class ProposalVotesController < Decidim::Proposals::ApplicationController
+      before_action :authenticate_user!
+
+      def create
+        @proposal = Proposal.find(params[:proposal_id])
+
+        set_votes_count_classes
+        set_vote_button_classes
+
+        @proposal.votes.create!(author: current_user)
```

Comment:
Should we use a command here? Also, we aren't handling errors (vote already exists, failed, etc)